lms/envs/common.py

```diff
@@ -3437,6 +3437,7 @@ def _make_locale_paths(settings):
 # Once a user has watched this percentage of a video, mark it as complete:
 # (0.0 = 0%, 1.0 = 100%)
 COMPLETION_VIDEO_COMPLETE_PERCENTAGE = 0.95
+COMPLETION_AGGREGATOR_BLOCK_TYPES = {'course', 'chapter', 'sequential', 'vertical'}
```

Comment:
Since the `completion_aggregator` app is a plugin app, then this settings change is handled by its [`plugin_settings`](https://github.com/open-craft/openedx-completion-aggregator/blob/208aaba043d9098360384c50decefabc4029ce56/completion_aggregator/settings/common.py#L12), so this is no longer needed?